### PR TITLE
[Improve][Connector-V2]modify config template

### PR DESCRIPTION
**Pull request by liugddx**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)

### CODE REVIEW

**Review by ashulin — COMMENTED** [review]
Now that connector v1 and v2 are not completely separated, for connector v1 we may still need to provide templates for each engine.
@hailin0 @EricJoy2048 @CalvinKirs CC

**Comment by liugddx** [comment]
> 
The configuration of V1 is no longer available and there is a problem with the configuration of V2.

**Comment by EricJoy2048** [comment]
> Now that connector v1 and v2 are not completely separated, for connector v1 we may still need to provide templates for each engine. @hailin0 @EricJoy2048 @CalvinKirs CC

@TyrantLucifer

**Comment by TyrantLucifer** [comment]
> Now that connector v1 and v2 are not completely separated, for connector v1 we may still need to provide templates for each engine.

Aggre with the opinion of @ashulin

**Comment by liugddx** [comment]
> > Now that connector v1 and v2 are not completely separated, for connector v1 we may still need to provide templates for each engine.
> 
> Aggre with the opinion of @ashulin
Ok, I temporarily rollback V1 config.

**Comment by liugddx** [comment]
@TyrantLucifer PTAL thanks.

File: config/batch.conf.template [hunk 1]
```diff
@@ -6,49 +6,58 @@
...
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
 ######
-###### This config file is a demonstration of streaming processing in SeaTunnel config
+###### This config file is a demonstration of streaming processing in seatunnel config
```
Comment (TyrantLucifer) [inline config/batch.conf.template]: `SeaTunnel` is better.

File: config/batch.conf.template [hunk 2]
```diff
@@ -6,49 +6,58 @@
...
 ######
 
 env {
   # You can set flink configuration here
```
Comment (TyrantLucifer) [inline config/batch.conf.template]:
```suggestion
  # You can set SeaTunnel environment configuration here
```

File: config/streaming.conf.template [hunk 3]
```diff
@@ -6,50 +6,58 @@
...
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
 ######
-###### This config file is a demonstration of batch processing in SeaTunnel config
+###### This config file is a demonstration of streaming processing in seatunnel config
```
Comment (TyrantLucifer) [inline config/streaming.conf.template]: `SeaTunnel` is better

File: config/streaming.conf.template [hunk 4]
```diff
@@ -6,50 +6,58 @@
...
 ######
 
 env {
   # You can set flink configuration here
```
Comment (TyrantLucifer) [inline config/streaming.conf.template]: The same as above.

**Review by TyrantLucifer — COMMENTED** [review]
BTW, change config file anme to `v2.batch.config.template`  `v2.streaming.config.template` is better. So user can know this template is below to SeaTunnel V2 connectors quickly. What do you think about it?

**Review by Hisoka-X — APPROVED** [review]

**Review by TyrantLucifer — APPROVED** [review]

**Pull request merged**